src/main/java/sirius/db/qdrant/QdrantDatabase.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.qdrant;
+
+import com.alibaba.fastjson.JSON;
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+import sirius.kernel.async.Operation;
+import sirius.kernel.commons.Tuple;
+import sirius.kernel.commons.Watch;
+import sirius.kernel.health.Exceptions;
+import sirius.kernel.settings.PortMapper;
+
+import javax.annotation.Nullable;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.List;
+
+/**
+ * Provides a client to access a qdrant vector database.
+ */
+public class QdrantDatabase {
+
+    private static final String QDRANT_DEFAULT_HOST = "qdrant";
+    private static final int QDRANT_DEFAULT_PORT = 6333;
+    private static final Duration CONNECT_TIMEOUT = Duration.ofSeconds(5);
+    private static final Duration DEFAULT_OPERATION_TIMEOUT = Duration.ofSeconds(60);
+    private static final String HEADER_CONTENT_TYPE = "Content-Type";
+    private static final String CONTENT_TYPE_APPLICATION_JSON = "application/json";
+    private static final int HTTP_STATUS_OK = 200;
```

Comment:
Aren't there common constants available for these?

Reply:
We commonly use constants from netty for this - but that's not available in -db - The status is available in HttpUrlConnection...